package.json

```diff
@@ -1,15 +1,16 @@
 {
-  "name": "@dream/team",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "name": "@dream-team/monorepo",
   "scripts": {
-    "prelint": "npm run prettier",
-    "lint": "npx turbo run lint"
+    "setup": "yarn install",
+    "setup:flat": "yarn install --flat",
+    "setup:pure": "yarn install --pure-lockfile",
+    "setup:update": "yarn install --update-checksums",
+    "clean:modules": "npx rimraf node_modules"
   },
   "workspaces": [
     "packages/*",
```

Comment:
Also, a quick question: Will Docker run this? If so, we 100% need to add Makefile or extend package.json